database/tiny/controller.mjs

```diff
@@ -0,0 +1,188 @@
+/**
+ * A TinyPEN Controller.  Actions here specifically fetch to the set TinyPEN API URLS.
+ * @see this.URLS
+ * 
+ * @author Bryan Haberberger
+ * https://github.com/thehabes 
+ */ 
+
+import fetch from 'node-fetch'
```

Comment:
Don't import this. Node 18+ already has fetch just built in. This library was a stopgap while it was being done.